CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
 # CHANGELOG
 
+## v0.37.7
+
+*June 27, 2024*
+
+This release contains a few bug fixes and performance improvements.
```

Comment:
I would write something like: `This release is centered around two topics: mempool and performance improvements. The mempool will stop accepting new transactions if the node can't keep up with rechecking the number of transactions already in the mempool. It also contains a few bug fixes.`

Reply:
basically being a little bit more descriptive

Reply:
Yes, thank you! I've added it to the changelog. I felt this too with the 0.38 release. I could've been more descriptive there too.